lib/auth/in_memory/Backend.js

```diff
@@ -164,8 +164,7 @@ class Backend {
 
     /**
     * Mocks Vault's response to a policy evaluation request
-    * Since policies not actually implemented in memory backend,
```

Comment:
Well, since there is no more support for users at all (not even legacy, but a log suggests to convert them to proper accounts: https://github.com/scality/Arsenal/pull/346/files/95734792a5ed177485565b7a3ef825332e4817bd#diff-4d2ac1e12369f170fdddd2b4f357da18R172), I think it's better to just drop the comment about users here.

Reply:
ah, so then this whole function is dead code. it only gets called if there is a user making a put bucket or multi-object delete call.